publish.js

```diff
@@ -54,6 +54,21 @@ publish.getCLIOpts = function () {
     return opts;
 };
 
+
+/**
+ * Returns the contents of a package.json file as JSON object
+ *
+ * @param moduleRoot {String} - the path to the root where the package.json is
+ *                              located. Will use process.cwd() by default.
+ * @returns {Object} - returns the contents of the package.json file as a JSON
+ *                     object.
+ */
+publish.getPkg = function (moduleRoot) {
```

Comment:
With this utility you may as well rewrite line 18 of this file to 
var pkg = publish.getPkg(__dirname)
and eliminate pkgPath
(together with a suitable reordering of the definitions in the file)